packages/components/src/Accordion/AccordionHeading.tsx

```diff
@@ -0,0 +1,24 @@
+import { ReactNode } from 'react';
+import { Heading } from 'react-aria-components';
+import { cn } from '@marigold/system';
+import { Button } from '../Button';
+import { ChevronRight } from '../icons/ChevronRight';
+import { useAccordionContext } from './AccordionContext';
+
+export interface AccordionHeaderProps {
+  title?: ReactNode;
+}
+
+export const AccordionHeader = ({ title }: AccordionHeaderProps) => {
```

Comment:
This should be a regular `children` not title attribute so that you can write:

`<Accordion.Header>Settings</Accordion.Header>`

Writing `<Accordion.Header title="Settings" />` feels a bit unnatural